persistence/block.go

```diff
@@ -36,43 +35,28 @@ func (p PostgresContext) GetLatestBlockHeight() (latestHeight uint64, err error)
 	return
 }
 
-// OPTIMIZE: evaluate if it's  faster to get this from the blockstore (or cache) than the SQL engine
-func (p PostgresContext) GetBlockHash(height int64) ([]byte, error) {
+func (p PostgresContext) GetBlockHash(height int64) (string, error) {
```

Comment:
Similar to @deblasis's previous comment, out of curiosity, what is the main reason to move to string from bytes?

Reply:
Same answer as here: https://github.com/pokt-network/pocket/pull/391#discussion_r1049075566

1. Simplicity - they're small, not big data blobs
2. Readability - Things like address or stateHash are "human readable" in the sense that we use & reference it (e.g. printing)
3. Performance (micro optimization) - lots of unnecessary encoding & decoding throughout the codebase